cmds/minigames/connect4.js

```diff
@@ -1,84 +1,141 @@
 const two_p_Game = require("../../typedefs/Game");
-const Myclient = require("../../typedefs/client")
+const Myclient = require("../../typedefs/client");
 
+class connect4 extends two_p_Game {
+    constructor(client) {
+        super(
+            "connect4",
+            "minigames",
+            client
+        );
 
 
-class connect4 extends two_p_Game{
-
-    constructor(client){
-
-        super("connect4","minigames",client)
-
-        this.image = "https://cdn.discordapp.com/attachments/737224846878179360/851102547603357746/connect_demo.gif"
-
-        this.cmd = async function(msg,args){
+        this.image =
+            "https://cdn.discordapp.com/attachments/737224846878179360/851102547603357746/connect_demo.gif";
 
+        this.cmd = async function (msg, args) {
             /**
-             * 
-             * @param {*} msg 
-             * @param {*} iuser 
-             * @param {Myclient} client 
+             * Create a connect 4 game
+             * @param {object} msg - Discord message object
+             * @param {object} iuser - User being invited to the game
+             * @param {Myclient} client - Client
              */
-  
-            async function connect4(msg,iuser,client){
 
-                let game = new c4(msg.author.id,iuser.id)
-                let end = false;
+            async function connect4(msg, iuser, client) {
+                // create a new connect 4 game between author and an invited user
+                let game = new c4(msg.author.id, iuser.id);
+                let end = false; // (unused variable?)
                 let current_color = "red";
                 let userlang = msg.author.lang
 
-
-                let field1 = {name: "Now : ",value: msg.author.username + client.language.commands.game[userlang].ones_turn , inline: true}
-                let rmsg = await msg.author.send(client.EmbedMaker(msg, game.stringify(), client.colors.red,[field1]))
-                let ymsg = await iuser.send(client.EmbedMaker(msg, game.stringify(), client.colors.red,[field1]))
-
-                for(var i = 0; i < 8; i++){
-                    rmsg.react(game.reactions[i])
-                    ymsg.react(game.reactions[i])
+                let field1 = {
+                    name: "Now : ",
+                    value: msg.author.username + client.language.commands.game[userlang].ones_turn,
```

Comment:
Language options like this is very lengthy.
Consider using 
```suggestion
                    let language = client.language.commands.game[userlang]
                    // and then in your code
                    value: msg.author.username + language.ones_turn,
```